Add statement for Dynamic Bundles

shaunmaharaj opened the pull request:
Description:
<!--A brief description of changes. Things to include: WIP? Dependent PR's opened against other tickets? -->
- Add statement regarding Dynamic Bundles

Linting:
<!--Have you validated that no linting errors are introduced? -->
- [x] No linting errors

Tests:
<!--Have tests been run locally and passed? If manual tests run, explain what was run below-->
- [ ] Smoke tests (mvn clean install -Dcucumber.options="--tags @smoketest" from test)
- [ ] Manual tests

Documentation:
<!--Are documentation updates required? Include any mention of updates required below if necessary-->
- [x] Requires documentation updates

sminnath commented on documentation/practices.md:
```diff
@@ -66,6 +66,8 @@ Google Analytics handlers use the global product attribute `Tag`. Use the format
 
 Featured Product handlers use the global product attribute `Featured`. Use the attribute's boolean value to indicate when a product is designated a featured product. You can set the global attribute for each product in the catalog within the Commerce Manager. The React PWA Reference Storefront uses the tag to display a customizable banner for the featured product.
 
+Dynamic Bundle Configuration uses the Elastic Path Accelerator for implementing Dynamic Bundles for products, which is distributed under a separate license. For more information, visit the Elastic Path Accelerators Github Repository upon successful activation of your Elastic Path license.
```
```suggestion
Use [Elastic Path Dynamic Bundle Accelerator](https://code.elasticpath.com/accelerators/dynamic-bundles) for implementing Dynamic Bundles for products. For more information about the accelerator license, see the License and Copyright page(https://www.elasticpath.com/sites/default/files/elastic-license-and-copyright.pdf).
```

shaunmaharaj commented:
@sminnath Addressed comments in latest push. Thank you for the review!
If there's no further comments/concerns, please approve the changes and I'll merge 😄
